### PR TITLE
Update nullable-reference-types.md

### PR DESCRIPTION
**Pull request by Youssef1313**

### CODE REVIEW

File: docs/csharp/tutorials/nullable-reference-types.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ description: This advanced tutorial provides an introduction to nullable referen
 ms.date: 02/19/2019
 ms.custom: mvc
 ---
-# Tutorial: Express your design intent more clearly with nullable and non-nullable reference types
+# Tutorial: express your design intent more clearly with nullable and non-nullable reference types
```
Comment (mairaw) [inline docs/csharp/tutorials/nullable-reference-types.md]: Even though we don't cover this scenario in our guidelines, I think that in the case of tutorials, I actually prefer if the title after the : starts with uppercase. It seems to be different than the How to: guidance that we were given.

File: docs/csharp/tutorials/nullable-reference-types.md [hunk 2]
```diff
@@ -59,7 +59,7 @@ If you've programmed in C#, you may be so accustomed to reference types that all
 
 As you write the code, you'll see that a non-nullable reference type as the default for references avoids common mistakes that could lead to null reference exceptions. One lesson from this tutorial is that you made decisions about which variables could or could not be null. The language didn't provide syntax to express those decisions. Now it does.
 
-The app you'll build will do the following steps:
+The app you'll build does the following steps:
 
 1. Create a survey and add questions to it.
```
Comment (mairaw) [inline docs/csharp/tutorials/nullable-reference-types.md]: It seems these steps should be in 3rd person, like
```suggestion
1. Creates a survey and adds questions to it.
```

**Comment by Youssef1313** [comment]
I've added a missing comma from #14890.

**Comment by Youssef1313** [comment]
@BillWagner, Please, can you review the text under ` ## Create the application and enable nullable reference types` ?

I suspect that it's outdated. With the release of C# 8.0, I don't think there is a need to enable the nullable manually by adding it to csproj file.

**Comment by BillWagner** [comment]
@Youssef1313 I just checked, and it is still necessary to explicitly opt in to a nullable context (either using `#nullable enable` in source code or `<Nullable>enable</Nullable>` in the project file.

**Comment by BillWagner** [comment]
I read through the changes, and I'm happy with this update.

@mairaw Do you have more feedback?
@Youssef1313 Is this ready for final review?

**Comment by mairaw** [comment]
Nope. changes LGTM.

**Comment by Youssef1313** [comment]
Made a few minor changes. It's now ready for final review, @BillWagner.

**Review by BillWagner — CHANGES_REQUESTED** [review]
This is looking very good.  I had one comment, related to the `LangVersion` and `Nullable` elements in the csproj file. I may not have expressed that well in my earlier comment. 

Once that's updated, this is ready to merge.

**Review by BillWagner — APPROVED** [review]

**Pull request merged**